gulpfile.js

```diff
@@ -87,24 +89,33 @@ gulp.task('buildCSS', function () {
     var sassCompilation = sass();
     sassCompilation.on('error', console.error.bind(console));
 
+    var processors = [
+        autoprefixer({browsers: ['last 4 versions']})
+    ];
+
     return gulp.src('./browser/scss/main.scss')
         .pipe(plumber({
             errorHandler: notify.onError('SASS processing failed! Check your gulp process.')
         }))
         .pipe(sassCompilation)
         .pipe(rename('style.css'))
+        .pipe(postcss(processors))
         .pipe(gulp.dest('./public'));
 });
 
 // Production tasks
 // --------------------------------------------------------------
 
 gulp.task('buildCSSProduction', function () {
+    var processors = [
+        autoprefixer({browsers: ['last 4 versions']})
+    ];
     return gulp.src('./browser/scss/main.scss')
         .pipe(sass())
         .pipe(rename('style.css'))
         .pipe(minifyCSS())
-        .pipe(gulp.dest('./public'))
+        .pipe(postcss(processors))
```

Comment:
@ldthorne could you just make sure I didn't screw up the gulp task somehow. I just added this line which just adds vendor prefixes to the CSS for better compatibility. I added this line both for the production gulp task and development gulp task